setup.py

```diff
@@ -15,9 +15,9 @@
 def post_install():
     path = '/usr/local/bin/woeusbgui'  # I give up, I have no clue how to get bin path that is used by pip
     shutil.copy2(this_directory + '/WoeUSB/woeusbgui', path)  # I'll just hard code it until someone finds better way
-
+    
```

Comment:
Please drop redundant spaces in blank lines.

Reply:
ok